stl/inc/xlocbuf

```diff
@@ -35,21 +35,26 @@ public:
     using off_type   = typename _Traits::off_type;
     using state_type = typename _Codecvt::state_type;
 
-    explicit wbuffer_convert(_Mysb* _Strbuf = nullptr)
-        : _State(), _Pcvt(new _Codecvt), _Mystrbuf(_Strbuf), _Status(_Unused),
-          _Nback(0) { // construct with byte stream buffer pointer
+    wbuffer_convert() : _State(), _Pcvt(new _Codecvt), _Mystrbuf(nullptr), _Status(_Unused), _Nback(0) {
+        // construct with byte stream buffer pointer

```

Comment:
This comment isn't especially accurate for the default constructor now.

Reply:
Yep, changed it to "without buffer pointer"